### PR TITLE
ZK-4742: client side http error handling not reachable

### PR DESCRIPTION
**Pull request by b952426**
ZK-4742: client side http error handling not reachable

### CODE REVIEW

File: zktest/test/java/org/zkoss/zktest/zats/test2/B95_ZK_4742Test.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Copyright (C) 2020 Potix Corporation. All Rights Reserved.
+*/
+package org.zkoss.zktest.zats.test2;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.zkoss.zktest.zats.WebDriverTestCase;
+
+
+public class B95_ZK_4742Test extends WebDriverTestCase {
```
Comment (scribetw) [inline zktest/test/java/org/zkoss/zktest/zats/test2/B95_ZK_4742Test.java:19]: @ClassRule
	public static final ExternalZkXml CONFIG = new ExternalZkXml(B95_ZK_4742Test.class);

**Review by scribetw — CHANGES_REQUESTED** [review]

File: zk/src/archive/web/js/zk/au.js [hunk 2]
```diff
@@ -1000,7 +1000,7 @@ zAu.beforeSend = function (uri, req, dt) {
 				}
 			}
 			if (!reqInf.ignorable && !zk.unloading) {
-				var msg = req.statusText;
+				var msg = reqInf.statusText;
```
Comment (cor3000) [inline zk/src/archive/web/js/zk/au.js:1003]: this is an even more important fix, we should also have a test case for the default client side error handling, besides the custom error-uri config

**Review by scribetw — APPROVED** [review]

**Review by plijyfes — APPROVED** [review]

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**